apps/dashboard/src/components/subscribers/create-subscriber-form.tsx

```diff
@@ -23,6 +23,7 @@ import TruncatedText from '../truncated-text';
 import { LocaleSelect } from './locale-select';
 import { CreateSubscriberFormSchema } from './schema';
 import { TimezoneSelect } from './timezone-select';
+import { useNavigateToSubscribersFirstPage } from './hooks/use-navigate-to-subscribers-first-page';
```

Comment:
Does this need to be a hook? Seems a bit excessive to have a hook for each navigation.

Maybe add all these to the `useSubscribersUrlState` hook ?